helm/starboard-exporter/values.yaml

```diff
@@ -86,3 +86,20 @@ customMetricsHPA:
   maxReplicas: 97  # The number of replicas is limited to 97 by the current sharding math. See https://www.giantswarm.io/blog/cveing-is-believing
   metricName: scrapedurationseconds
   targetAverageValueSeconds: 10  # Scrape duration seconds timeout
+
+scaledObject:
+  enabled: false
+  minReplicas: 2
+  maxReplicas: 97
```

Comment:
Since we now have these same values in HPA and KEDA, can we write them once and use anchors for the individual values?

Reply:
It would look like this:

```
customMetricsHPA:
  enabled: false
  minReplicas: &minReplicas 2
  maxReplicas: &maxReplicas 97  # The number of replicas is limited to 97 by the current sharding math. See https://www.giantswarm.io/blog/cveing-is-believing

kedaScaledObject:
  enabled: false
  minReplicas: *minReplicas
  maxReplicas: *maxReplicas
```

Reply:
Either that or moving the values themselves out entirely:
```
minReplicas: &minReplicas 2
maxReplicas: &maxReplicas 97  # The number of replicas is limited to 97 by the current sharding math. See https://www.giantswarm.io/blog/cveing-is-believing

customMetricsHPA:
  enabled: false
  minReplicas: *minReplicas
  maxReplicas: *maxReplicas

kedaScaledObject:
  enabled: false
  minReplicas: *minReplicas
  maxReplicas: *maxReplicas
```

Reply:
Bam